src/main/java/core/basesyntax/db/DataBase.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataBase {
```

Comment:
```suggestion
public class Database {
```
filename, database - it's just one word